src/Microsoft.Azure.WebJobs.Extensions.Kafka/Output/KafkaAttribute.cs

```diff
@@ -75,13 +75,44 @@ public Type ValueType
         /// </summary>
         public string AvroSchema { get; set; }
 
-        bool IsValidValueType(Type value)
+        private bool IsValidValueType(Type value)
         {
             return
                 typeof(ISpecificRecord).IsAssignableFrom(value) ||
                 typeof(Google.Protobuf.IMessage).IsAssignableFrom(value) ||
                 value == typeof(byte[]) ||
                 value == typeof(string);
         }
+
+        /// <summary>
+        /// Gets or sets the Maximum transmit message size. Default: 1MB
+        /// </summary>
+        public int MaxMessageBytes { get; set; } = 1024000;
```

Comment:
What about making this fields nullable as they are in Confluent.Kafka?

The problem I see is that they (Confluent) have a better understand of what the default values should be. If we just take a snapshot of the current values it will be easy to miss improvements/changes they make in their repo.

That's the approach I have taken with the consumer settings.

Reply:
Yeah I flip-flopped on this before choosing one. Happy to change it!

Reply:
Do you think the default (as stated in the current SDK's docs) should be part of the comment? I kinda feel like it should for a .Net dev... but realize it may get out of sync in the future... thoughts?

Reply:
Agree. I have them commented in both documentation and KafkaOptions class, even though they could get out of sync.